src/main/java/org/springframework/retry/annotation/AnnotationAwareRetryOperationsInterceptor.java

```diff
@@ -126,12 +126,12 @@ public void setNewItemIdentifier(NewMethodArgumentsIdentifier newMethodArguments
 
 	/**
 	 * Retry listeners to apply to all operations.
-	 * @param listeners the listeners
+	 * @param defaultListeners the listeners
 	 */
-	public void setListeners(Collection<RetryListener> listeners) {
-		ArrayList<RetryListener> retryListeners = new ArrayList<RetryListener>(listeners);
+	public void setDefaultListeners(Collection<RetryListener> defaultListeners) {
```

Comment:
This is an API change, so it's hard to justify. I would think we can leave the method name as it is (if I understand the feature correctly) and document it better? IMO this should be a list of "global" interceptors that apply to all `@Retryables``.

Reply:
Ok. Makes sense.

Reply:
Changed back the method name, not to break API